src/compute.rs

```diff
@@ -130,6 +130,18 @@ pub fn poly(x: &Scalar, f: &Vec<Point>) -> Result<Point, PointError> {
     Point::multimult(s, f.clone())
 }
 
+/// Evaluate the private polynomial `f` at scalar `x`
+#[allow(clippy::ptr_arg)]
+pub fn private_poly(x: Scalar, f: &Vec<Scalar>) -> Scalar {
```

Comment:
I've been using `params` to refer to the coefficients, mostly because `coeffs` doesn't parse as well and they mean (basically) the same thing.  

But the good news is that we can get rid of this `fn` entirely, since `common::Polynomial` is working and in place.

Reply:
Done